### PR TITLE
Update jck23 lang class group 

### PR DESCRIPTION
**Pull request by sophia-guo**
Part of #5572 
Update jck  lang class group

### CODE REVIEW

**Comment by sophia-guo** [comment]
lang.CLSS
   lang.EXPR
   lang.CONV
   lang.LMBD
   lang.DASG
   lang.NAME
   lang.TYPE
   lang.STMT

**Review by karianna — APPROVED** [review]

**Review by andrew-m-leonard — APPROVED** [review]

File: jck/subdirs/jck23.mk [hunk 1]
```diff
@@ -12,7 +12,7 @@ COMPILER_LANG_CLASS_TESTS_GROUP6=$(Q)lang/CLSS/clss329;lang/CLSS/clss333;lang/CL
...
+COMPILER_LANG_CLASS_TESTS_GROUP10=$(Q)lang/CLSS/clss541;lang/CLSS/clss542;lang/CLSS/clss543;lang/CLSS/clss544;lang/CLSS/clss545;lang/CLSS/clss546;lang/CLSS/clss547;lang/CLSS/clss548;lang/CLSS/clss549;lang/CLSS/clss551;lang/CLSS/clss552;lang/CLSS/clss553;lang/CLSS/clss554;lang/CLSS/clss555;lang/CLSS/clss556;lang/CLSS/clss557;lang/CLSS/clss558;lang/CLSS/clss559;lang/CLSS/clss560;lang/CLSS/clss561;lang/CLSS/clss562;lang/CLSS/clss563;lang/CLSS/clss564;lang/CLSS/clss565;lang/CLSS/clss566;lang/CLSS/clss567;lang/CLSS/clss568;lang/CLSS/clss569;lang/CLSS/clss570;lang/CLSS/clss571;lang/CLSS/clss572;lang/CLSS/clss573;lang/CLSS/clss574;lang/CLSS/clss575;lang/CLSS/clss576;lang/CLSS/clss577;lang/CLSS/clss578;lang/CLSS/clss579;lang/CLSS/clss580;lang/CLSS/clss581lang/CLSS/clss582;lang/CLSS/clss583$(Q)
```
Comment (llxia) [inline jck/subdirs/jck23.mk:15]: `;` is missing after `clss581lang`

**Comment by llxia** [comment]
Please resolve the merge conflict.

**Review by karianna — CHANGES_REQUESTED** [review]
Rebase req

**Review by karianna — APPROVED** [review]

**Pull request merged**